lib/dkim.js

```diff
@@ -463,8 +464,9 @@ function DKIMVerify(email, callback, sigs) {
 
         // The other DNS errors are operational in nature;
         sigs.result = sig.result = false;
-        sigs.issue_desc = sig.issue_desc = err;
-        return callback(err, sigs);
+        sigs.issue_desc = sig.issue_desc = err.message;
+
+        return callback(null, sigs); // returning the error AND the sigs breaks promisification... should do callback(null, sigs)
```

Comment:
Ha ok just so I understand -- does it break something in how promises work somehow or does it just continue to throw an error that we actually want to swallow? I think it's the latter which may just be a semantic difference in how we're describing this but wanted to make sure I understood what was happening. :)